src/renderer/components/status-bar/auto-update-component.tsx

```diff
@@ -28,51 +29,39 @@ interface EndNoteProps {
   note: (version: string) => JSX.Element;
 }
 
-const EndNote = ({ version, note }: EndNoteProps) => {
-  const [idling, setIdling] = useState(false);
+const EndNote = observer(({ version, note }: EndNoteProps) => {
+  const [start] = useState(Date.now());
 
-  useEffect(() => {
-    const timerId = setTimeout(() => setIdling(true), 5000);
-
-    return () => clearTimeout(timerId);
-  });
-
-  if (idling) {
+  if (start + 5000 <= reactiveDateNow()) {
     return idle();
   }
-
+  
   return note(version ?? "");
-};
-
-const DivWithTestId = (text: string) => <div data-testid="auto-update-component">{text}</div>;
+});
```

Comment:
modified this at @Nokel81's suggestion, in an attempt to get the behavioural test to work.

Reply:
Cool. :)